Makefile

```diff
@@ -7,24 +7,35 @@ all: build
 .ONESHELL:
 install:
 	if hash apt-get 2>/dev/null; then
-	  sudo apt-get update -qq >/dev/null && sudo apt-get install -qq apt-utils imagemagick python3-pip python3-setuptools gcc
+	  sudo apt-get update -qq >/dev/null && sudo apt-get install -qq apt-utils imagemagick python3-pip python3-setuptools gcc git-lfs
 	elif hash pacman 2>/dev/null; then
-	  sudo pacman -Syu imagemagick python-pip glibc lib32-glibc gcc --noconfirm
+	  sudo pacman -Syu imagemagick python-pip glibc lib32-glibc gcc git-lfs --noconfirm
 	elif hash dnf 2>/dev/null; then
-	  sudo dnf install -y ImageMagick python3-pip gcc
+	  sudo dnf install -y ImageMagick python3-pip gcc git-lfs
 	else
-	  echo -e "Please install Imagemagick, python3-pip and gcc"
+	  echo -e "Please install Imagemagick, python3-pip git-lfs and gcc"
 	fi
-	pip3 install --user --upgrade wheel
-	pip3 install --user --upgrade lektor
+	sudo pip3 install wheel  --upgrade
+	sudo pip3 install lektor --upgrade
 
 build:
-	lektor clean --yes
-	lektor plugin flush-cache
-	lektor build $(LEKTOR_PLUGIN_FLAGS)
+	if python3 -m lektor --version 2>/dev/null; then
```

Comment:
If we use the system's package installer we probably do not need the `python3 -m` calls. Also Python 3.x might not be called `python3` on every system, depending on whether the distribution uses it as a default instead of Python 2.x

Reply:
Daher ist das ja in einer extra if abfrage drinnen.

Sonst haben wir auf altertümlichen systemen das problem das der python2 build von lektor verwended wird der das sass plugin nicht korrekt unterstützt!